apps/meteor/server/models/raw/Messages.ts

```diff
@@ -1310,6 +1320,7 @@ export class MessagesRaw extends BaseRaw<IMessage> implements IMessagesModel {
 
 		if (!limit) {
 			const count = (await this.deleteMany(query)).deletedCount;
+			await ReadReceipts.removeByIdPinnedTimestampLimitAndUsers(rid, pinned, ignoreDiscussion, ts, users, ignoreThreads);
 
 			// decrease message count
 			await Rooms.decreaseMessageCountById(rid, count);
```

Comment:
I know you'll see this and will think that I'm crazy since we're already using models inside of models.. but this is legacy code and is considered technical debt that should be removed.

Reply:
I completely understand hahah I've just removed the `ReadReceipts` model import from inside the `Messages` model.